server/src/test/resources/logback-test.xml

```diff
@@ -13,4 +13,5 @@
     </root>
 
     <logger name="org.flywaydb" level="ERROR"/>
+<!--    <logger name="io.micronaut.data" level="DEBUG"/>-->
```

Comment:
Commented out line may be removed if not needed.

Reply:
Normally I would agree with you, but Michael likes to leave this one as it is often, but not always, needed and this way it is there when we do need it. =)